### DISCUSSION TITLE
Documentation for Admin

### DISCUSSION THREAD

**Discussion by guidiego** [prest/prest#571] [Admin Frontend]
## Description
As a new product, we need to create minimal documentation to made pRest.admin easy to be set on any user environment.

**Comment by worais** [comment]
If you need I can help.

**Comment by avelino** [comment]
> If you need I can help.

Very good @geekdoagrest, @guidiego updates the issue how you think about documenting and what he recommends to use to write the documentation.

I recommend staying inside the official documentation of the project, so we would use Hugo, [more here](https://github.com/prest/prest/tree/master/docs)

**Comment by worais** [comment]
@avelino I already worked with Hugo in tabless website. it's very quiet...

**Comment by guidiego** [comment]
Yeah! I agree, add the doc inside project will be the better approach! 

About Technologie, I'd like to use Docusaurs, it's a Facebook documentation lib that use react to customize templates and this kind of things. Like Hugo, we use .md to populate our documentation. But I am open to discuss about it!

**Comment by worais** [comment]
@guidiego I looked at the docusaurus documentation and it seems easy, if it is not a problem to use something other than the official documentation, I agree.

**Comment by guidiego** [comment]
@geekdoagrest I've used Hugo before and doesn't see any problems in use it! My point with Docusaurus is:

- We could add it on package.json and not install more deeps to run
- More friendly to edit templates (talking about React developers)
- I really like Docusaurus flexibility

Also, I 100% confident that if you prefer use Hugo you will make a good job! So feel free to take this decision and I will be glad to review your PR! 🥳

**Comment by worais** [comment]
@guidiego ok, I also think the Docusaurus is the best way. Commit this repo or https://github.com/prest/prest.admin?

**Comment by guidiego** [comment]
@geekdoagrest in prest.admin please! Here we will only tracking issues :)

**Comment by c-nv-s** [comment]
I'm trying to build the documentation locally for offline use. What are the commands to build it please?
